sos/policies/redhat.py

```diff
@@ -191,6 +190,10 @@ def get_tmp_dir(self, opt_tmp_dir):
 
 RHOCP = "ocp"
 RHOCP_DESC = "OpenShift Container Platform by Red Hat"
+RHOSP_OPTS = SoSOptions(plugopts=[
+                             'process.lsof=off',
+                             'networking.ethtool_namespaces=False',
+                             'networking.inspect_namespaces=200'])
```

Comment:
Needs to be updated to just `networking.namespaces=200`

Reply:
Updated in latest commit.